isis/tests/Fixtures.h

```diff
@@ -121,6 +121,26 @@ namespace Isis {
       void TearDown() override;
   };
 
+
+  class StereoPair : public TempTestingFiles {
```

Comment:
Be careful, this isn't a stereo pair. This is just a Left & Right LRO NAC observation. So, they are captured simultaneously and from very similar viewing angles. I would call this like LRONACPair or ObservationPair.